server/server.js

```diff
@@ -276,9 +244,21 @@ UserPresence = {
 				}
 			};
 
-			Meteor.users.update(query, update);
+			const result = Meteor.users.update(query, update);
 
-			UserPresenceEvents.emit('setUserStatus', user, status, statusConnection);
+			// if nothing updated, do not stream anything
+			if (result) {
```

Comment:
why not just emit an event and let the application deals with the situation? I don't like that meteor-streamer, so we can use any package we want: x

Reply:
good question

Reply:
streamer removed =)